quipucords/api/source/tests_source.py

```diff
@@ -937,6 +972,27 @@ def test_update_vc_more_than_one_host(self):
                                    format='json')
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
 
+    def test_update_vc_with_exclude_host(self):
+        """VC - Fail more than one host."""
```

Comment:
This docstring should say something more along the lines of """VC - Fail with exclude hosts."""